src/ui/utils/telemetry.ts

```diff
@@ -1,19 +1,18 @@
-import mixpanel from "mixpanel-browser";
 import * as Sentry from "@sentry/react";
 import { Integrations } from "@sentry/tracing";
-import { isDevelopment, skipTelemetry } from "./environment";
+import { skipTelemetry } from "./environment";
 import { Recording, Workspace } from "ui/types";
-import { isTest } from "./environment";
 import { prefs } from "./prefs";
 import { UserInfo } from "ui/hooks/users";
+import { initializeMixpanel, trackMixpanelEvent } from "./mixpanel";
 
-let mixpanelDisabled = false;
 const timings: Record<string, number> = {};
-const QA_EMAIL_ADDRESSES = ["mock@user.io"];
 
 export function setupTelemetry() {
   const ignoreList = ["Current thread has paused or resumed", "Current thread has changed"];
-  mixpanel.init("ffaeda9ef8fb976a520ca3a65bba5014");
+  // We always initialize mixpanel here. This allows us to force enable mixpanel events even if
+  // telemetry events are being skipped for any reason, e.g. development, test, etc.
+  initializeMixpanel();
```

Comment:
I'm a bit confused about why we initialize it if we never plan on calling it, but I'm sure there are Reasons™️